From 46c0d86b8bbae226e041f867dd2f067c45b634bd Mon Sep 17 00:00:00 2001
From: Raffaele Aquilone <raquiloX>
Date: Fri, 11 Jan 2013 17:46:55 +0100
Subject: [PORT FROM MAIN][BT] Broadcom chip SCO PCM configuration

BZ: 79880

Changing the clock rate in the SCO PCM configuration of the
Broadcom BT chip to the lowest one available (0 = 128k), it was
tested and proved to be working

BT chip needs also to be master on PCM SCO link with audio codec

Change-Id: I6fd82d3753b285722b9e8a0e21d73279b04eb005
Orig-Change-Id: Ia7dc6ce4cbb0cd920d57959e2ea74ffd7d76a9f5
Feature: BT
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: no
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/86305
Reviewed-by: Bellouard, Jean-marie <jean-marie.bellouard@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libbt/include/bt_vendor_brcm.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/libbt/include/bt_vendor_brcm.h b/libbt/include/bt_vendor_brcm.h
index 858b71b..2989c50 100644
--- a/libbt/include/bt_vendor_brcm.h
+++ b/libbt/include/bt_vendor_brcm.h
@@ -283,7 +283,7 @@
     4 : 2048K
 */
 #ifndef SCO_PCM_IF_CLOCK_RATE
-#define SCO_PCM_IF_CLOCK_RATE           4
+#define SCO_PCM_IF_CLOCK_RATE           0
 #endif
 
 /* SCO_PCM_IF_FRAME_TYPE - 0=Short, 1=Long */
@@ -293,12 +293,12 @@
 
 /* SCO_PCM_IF_SYNC_MODE - 0=Slave, 1=Master */
 #ifndef SCO_PCM_IF_SYNC_MODE
-#define SCO_PCM_IF_SYNC_MODE            0
+#define SCO_PCM_IF_SYNC_MODE            1
 #endif
 
 /* SCO_PCM_IF_CLOCK_MODE - 0=Slave, 1=Master */
 #ifndef SCO_PCM_IF_CLOCK_MODE
-#define SCO_PCM_IF_CLOCK_MODE           0
+#define SCO_PCM_IF_CLOCK_MODE           1
 #endif
 
 #define PCM_DATA_FORMAT_PARAM_SIZE      5
-- 
1.7.4.1

